sanic/request.py

```diff
@@ -7,7 +7,11 @@
 try:
     from ujson import loads as json_loads
 except ImportError:
-    from json import loads as json_loads
+    from json import loads as _json_loads
+
+    def json_loads(i):
+        i = i.decode() if isinstance(i, bytes) else i
+        return _json_loads(i)
```

Comment:
Yes, but I mean it's safe to always run `i.decode()` since `i` will always be `bytes`.

```python
def json_loads(i):
    return _json_loads(i.decode())
```